packages/avocode-email-tagsinput/test/e2e/unique.spec.js

```diff
@@ -0,0 +1,57 @@
+import puppeteer from 'puppeteer'
+import { expect } from 'chai'
+import {
+  URL,
+  tagsInputFieldSelector,
+  addEmailTag,
+  assertEmailTagCount,
+} from './test-utils'
+
+describe('Unique Email Tags Input', () => {
+  let browser
+  let page
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch(browserOptions)
+  })
+
+  afterAll(async () => {
+    await browser.close()
+  })
+
+  beforeEach(async () => {
+    page = await browser.newPage()
+    await page.goto(`${URL}/unique`)
+  })
+
+  afterEach(async () => {
+    await page.close()
+  })
+
+  it('should render editor', async () => {
+    const editor = await page.waitForSelector(tagsInputFieldSelector)
+    expect(editor).to.exist
+  })
+
+  describe('user typing different email edge cases', () => {
+    const uniqueEmailExamples = [
+      "single'123@quote.aa",
+      'withcu{}ly@bracket.go',
+    ]
+
+    it('should be allowed to add unique email tags', async () => {
+      let tagCount = 0
+      for (const testEmail of uniqueEmailExamples) {
```

Comment:
Delete dynamic specs as explained in other comments. Just add a single spec where you're adding unique emails and that's it.